CHANGELOG.asciidoc

```diff
@@ -21,6 +21,23 @@ endif::[]
 [[release-notes-3.x]]
 === Node.js Agent version 3.x
 
+[[release-notes-3.8.0]]
+==== 3.8.0 - 2020/11/09
+
+[float]
+===== Features
+
+* feat: expand k8s pod ID discovery regex {pull}1863[#1863]
+* feat: implements tracestate {pull}1828[#1828]
+* feat: add span and transaction outcome {pull}1828[#1828]
```

Comment:
@astorm Would you be willing to suggest some prose on the tracestate addition? I can take a stab at something for "outcome".